src/main/java/org/opensearch/knn/index/KNNClusterContext.java

```diff
@@ -49,20 +46,11 @@ public void initialize(final ClusterService clusterService) {
      */
     public Version getClusterMinVersion() {
         Version minVersion = Version.CURRENT;
```

Comment:
```suggestion
        public Version getClusterMinVersion() {
            try {
                return this.clusterService.state().getNodes().getMinNodeVersion();
            } catch (Exception exception) {
                log.error("Failed to get minimum node version of the cluster. Returning current node version instead.", exception);
                return Version.CURRENT;
            }
        }
```

Reply:
Trivial. Better not to change variable value once it is defined to reduce a chance of an error.

Reply:
ack